process-engine/management_api_meta
# Fix integration tests

Depends on:
* https://github.com/process-engine/management_api_contracts/pull/16
* https://github.com/process-engine/management_api_core/pull/12
* https://github.com/process-engine/management_api_http/pull/10
* https://github.com/process-engine/management_api_client/pull/11
* https://github.com/process-engine/process_engine/pull/144/files
* https://github.com/process-engine/process_engine_contracts/pull/58
* https://github.com/process-engine/deployment_api_core/pull/2

## What did you change?

* changed `updateProcessModelById` to `updateProcessDefinitionsByName` in `update_process_definitions_by_name.js`
* updated dependencies to feature branches that need to be tested

## How can others test the changes?

All integration tests in meta repositories (including this one) should run successfully: 

* https://github.com/process-engine/consumer_api_meta/pull/40
* https://github.com/process-engine/deployment_api_meta/pull/3
* https://github.com/process-engine/process_engine_meta/pull/122

## PR-Checklist

Please check the boxes in this list after submitting your PR:
